src/lib/MapLibre.svelte

```diff
@@ -24,8 +25,8 @@
    * make minimal changes. If you enable this, be aware of https://github.com/maplibre/maplibre-gl-js/issues/2651,
    * which may prevent some style changes from becoming visible when diffing is enabled. */
   export let diffStyleUpdates = false;
-  export let center: LngLatLike = [0, 0];
-  export let zoom = 1;
+  export let center: LngLatLike | undefined = undefined;
```

Comment:
I'm fairly sure this + the flush should be behaviorally equivalent to what's happening now. The default center and zoom in maplibre is at https://github.com/maplibre/maplibre-gl-js/blob/5d7e6d52000a8569ac2308a9aef14c98933eb0d8/src/ui/map.ts#L370. The only difference is defaulting to 0 zoom, vs 1 here. I'm not sure if this PR should be considered a breaking API change and warrant a major version bump.